firmware/src/src/sense.cpp

```diff
@@ -920,6 +937,7 @@ void rotate(float pn[3], const float rotation[3])
 
 void reset_fusion()
 {
+  // TODO add a mutex here.
```

Comment:
hmm, this has been this way for a while -- i guess since this is threaded?

Reply:
Yeah as I was going through it I just noticed that. Oh hey this has some variables used in multiple threads.